### PR TITLE
feat: Contribute i18n swedish language

### PR DESCRIPTION
**Pull request by JesperWe**
# Which Problems Are Solved

- Swedish speakers cannot use their beautiful native language ;-)

# How the Problems Are Solved

- Contributes Swedish language for Login, Console, common texts and Emails

# Additional Changes

- none

# Additional Context

- The PR currently provides all translation files according to https://github.com/zitadel/zitadel/blob/main/CONTRIBUTING.md#contribute-internationalization.

### CODE REVIEW

File: internal/api/ui/login/static/i18n/sv.yaml [hunk 1]
```diff
@@ -0,0 +1,500 @@
...
+  OTPDescription: Scanna koden med din autentiserings-app (T ex Google/Microsoft Authenticator, Authy). Eller kopiera nyckeln och fyll i den genererade säkerhetskoden här nedan.
+  SecretLabel: Nyckel
+  CodeLabel: Säkerhetskod
+  NextButtonText: Fortsätt
+  CancelButtonText: Avbryt
+
+InitMFAOTPSMS:
+  Title: Tvåfaktor verifiering
+  DescriptionPhone: Verifiera med en andra faktor. Ange ditt mobilnummer.
+  DescriptionCode: Verifiera med en andra faktor. Ange koden du fått på SMS för att bekräfta till mobilnr.
```
Comment (bohemima) [inline internal/api/ui/login/static/i18n/sv.yaml:111]: bekräfta _ditt_ mobilnr

File: internal/api/ui/login/static/i18n/sv.yaml [hunk 2]
```diff
@@ -0,0 +1,500 @@
...
+  TosAndPrivacyLabel: Användarvillkor
+  TosConfirm: Jag accepterar
+  TosLinkText: Användarvillkoren
+  PrivacyConfirm: Jag accepterar
+  PrivacyLinkText: personuppgiftspolicyn
+  SaveButtonText: Skapa organisation
+
+LoginSuccess:
+  Title: Inloggad
+  AutoRedirectDescription: Duy kommer automatiskt skickas vidare till tjänsten. Om inget händer kan du klicka på länken nedan. Sedan kan du stänga det här fönstret.
```
Comment (bohemima) [inline internal/api/ui/login/static/i18n/sv.yaml:322]: Duy

File: internal/api/ui/login/static/i18n/sv.yaml [hunk 3]
```diff
@@ -0,0 +1,500 @@
...
+  NextButtonText: Fortsätt
+
+LogoutDone:
+  Title: Utloggad
+  Description: Du har nu loggats ut.
+  LoginButtonText: Logga in igen
+
+LinkingUserPrompt:
+  Title: Det finns redan ett konto
+  Description: "Vill du kopla ihop din inoggning med det befintliga kontot:"
```
Comment (bohemima) [inline internal/api/ui/login/static/i18n/sv.yaml:333]: inoggning

File: internal/api/ui/login/static/i18n/sv.yaml [hunk 4]
```diff
@@ -0,0 +1,500 @@
...
+      NotFound: Koden hittades inte
+      GeneratorAlgNotSupported: Genereringsalgoritmen stöds inte
+    EmailVerify:
+      UserIDEmpty: UserID är tom
+    ExternalData:
+      CouldNotRead: Externa data kunde inte läsas rätt
+    MFA:
+      NoProviders: Det finns inga MFA-leverantörer tillgängliga
+      OTP:
+        AlreadyReady: Multifactor OTP (OneTimePassword) är redan konfigurerat
```
Comment (bohemima) [inline internal/api/ui/login/static/i18n/sv.yaml:471]: Ibland "multifaktor" och ibland tvåfaktor, kan vara ok dock

**Review by bohemima — CHANGES_REQUESTED** [review]
Left a few comments, noticed some of them are in Swedish :)

File: internal/api/ui/login/static/i18n/sv.yaml (hunk 4 above)
Comment (bohemima) [reply internal/api/ui/login/static/i18n/sv.yaml:471]: Frågan är om multifaktor ska skrivas med k istället för c här också btw

**Comment by hifabienne** [comment]
Hei @JesperWe 
thanks for your contribution we really appreciate it.
I quickly searched for an older pr which contributed a new language, so you can see what files have to be changed: https://github.com/zitadel/zitadel/pull/6870/files

I will also update the contribution guidelines with the missing files. 

Please also note that we are using semantic release, this means the pr title needs to include the what it is about feat: docs: fix, etc. i will change it in the title, just as information in case you will contribute some more in the future :)

File: internal/api/ui/login/static/i18n/sv.yaml (hunk 4 above)
Comment (JesperWe) [reply internal/api/ui/login/static/i18n/sv.yaml:471]: Kollade upp lite. 

https://www.digg.se/digitala-tjanster/e-legitimering/tillitsnivaer-for-e-legitimering

Vi ska nog hålla det till tvåfaktor. "Multifaktor" verkar inte vara vedertaget.

File: internal/api/ui/login/static/i18n/sv.yaml (hunk 3 above)
Comment (JesperWe) [reply internal/api/ui/login/static/i18n/sv.yaml:333]: Ja, och två p i koppla :-D

File: internal/api/ui/login/static/i18n/sv.yaml [hunk 5]
```diff
@@ -0,0 +1,500 @@
...
+  Provider1: Din fysiska mobil/laptop (T ex FaceID, Windows Hello, Fingeravtryck)
+  Provider3: Engångslösenord på SMS
+  Provider4: Engångslösenord på E-Post
+  NextButtonText: Fortsätt
+  SkipButtonText: Använd inte 2-faktor
+
+InitMFAOTP:
+  Title: Tvåfaktor verifiering
+  Description: Bekräfta med en andra faktor. Ladda ned en autentiserings-app (T ex Google/Microsoft Authenticator, Authy) om du inte redan har en.
+  OTPDescription: Scanna koden med din autentiserings-app (T ex Google/Microsoft Authenticator, Authy). Eller kopiera nyckeln och fyll i den genererade säkerhetskoden här nedan.
```
[the remark this answers is not in the document]
Comment (JesperWe) [reply internal/api/ui/login/static/i18n/sv.yaml:102]: https://svenska.se/saol/?sok=scannar

**Comment by JesperWe** [comment]
@hifabienne I have followed the czech template and updated accordingly.

File: internal/static/i18n/sv.yaml [hunk 6]
```diff
@@ -0,0 +1,1343 @@
...
+        set: Användarmetadata inställd
+        removed: Användarmetadata borttagen
+        removed.all: All användarmetadata borttagen
+      domain:
+        claimed: Domän anspråkad
+        claimed.sent: Meddelande om anspråkad domän skickad
+      pat:
+        added: Personlig åtkomsttoken tillagd
+        removed: Personlig åtkomsttoken borttagen
+  org:
```
Comment (muhlemmer) [inline internal/static/i18n/sv.yaml:835]: Wrong indentation level. The following nodes belong to the `user` not `user.human`. Resulted in a yaml error.

image

```suggestion
    locked: Användare låst
    unlocked: Användare upplåst
    deactivated: Användare avaktiverad
    reactivated: Användare återaktiverad
    removed: Användare borttagen
    password:
      changed: Lösenord ändrat
      code:
        added: Lösenordskod genererad
        sent: Lösenordskod skickad
      check:
        succeeded: Lösenordskontroll lyckades
        failed: Lösenordskontroll misslyckades
    phone:
      changed: Mobilnummer ändrat
      verified: Mobilnummer verifierat
      verification:
        failed: Verifiering av Mobilnummer misslyckades
      code:
        added: Mobilnummerkod genererad
        sent: Mobilnummerkod skickad

    profile:
      changed: Användarprofil ändrad
    address:
      changed: Användaradress ändrad
    mfa:
      otp:
        added: Tvåfaktor OTP tillagd
        verified: Tvåfaktor OTP verifierad
        removed: Tvåfaktor OTP borttagen
        check:
          succeeded: Tvåfaktor OTP-kontroll lyckades
          failed: Tvåfaktor OTP-kontroll misslyckades
        init:
          skipped: Tvåfaktor OTP-initialisering hoppades över
      init:
        skipped: Tvåfaktorinitialisering hoppades över
    signed:
      out: Användare utloggad
    grant:
      added: Auktorisering tillagd
      changed: Auktorisering ändrad
      removed: Auktorisering borttagen
      deactivated: Auktorisering avaktiverad
      reactivated: Auktorisering återaktiverad
      reserved: Auktorisering reserverad
      released: Auktorisering släppt
      cascade:
        removed: Auktorisering borttagen
        changed: Auktorisering ändrad
    metadata:
      set: Användarmetadata inställd
      removed: Användarmetadata borttagen
      removed.all: All användarmetadata borttagen
    domain:
      claimed: Domän anspråkad
      claimed.sent: Meddelande om anspråkad domän skickad
    pat:
      added: Personlig åtkomsttoken tillagd
      removed: Personlig åtkomsttoken borttagen
```

File: internal/api/ui/login/static/i18n/sv.yaml [hunk 7]
```diff
@@ -0,0 +1,500 @@
...
+  Description: Verifiera med kod från din Tvåfaktor-enhet
+  CodeLabel: Kod
+  ResendButtonText: Skicka ny kod
+  NextButtonText: Fortsätt
+
+VerifyMFAU2F:
+  Title: Verifiera tvåfaktor
+  Description: Verifiera med kod från din Tvåfaktor-enhet (t ex FaceID, Windows Hello, Fingeravtryck)
+  NotSupported: Din webbläsare har inte stöd för WebAuthN-standarden. Kontrollera att du har den senaste versionen, eller byt till en webbläsare med stöd (t ex Chrome, Safari, Firefox)
+  ValidateTokenButtonText: Verifiera tvåfaktor
```
Comment (muhlemmer) [inline internal/api/ui/login/static/i18n/sv.yaml:156]: Missing translation:

https://github.com/jesperwe/zitadel/blob/f1f7b12c5ed4132da7874ae3ef7e7198d0e5cc47/internal/api/ui/login/static/i18n/en.yaml#L156

**Review by muhlemmer — CHANGES_REQUESTED** [review]
Thanks for the PR, found some small errors.

File: internal/api/ui/login/static/i18n/sv.yaml [hunk 8]
```diff
@@ -0,0 +1,500 @@
...
+UsernameChange:
+  Title: Ändra användarnamn
+  Description: Ange ditt nya användarnamn
+  UsernameLabel: Användarnamn
+  CancelButtonText: Avbryt
+  NextButtonText: Fortsätt
+
+UsernameChangeDone:
+  Title: Användarnamn ändrat
+  Description: Ditt användarnamn har ndrats.
```
[the remark this answers is not in the document]
Comment (JesperWe) [reply internal/api/ui/login/static/i18n/sv.yaml]: Fixed

File: internal/static/i18n/sv.yaml [hunk 9]
```diff
@@ -0,0 +1,1343 @@
...
+    NotAllowedOrg: Användare är inte medlem i den nödvändiga organisationen
+    UserIDMissing: Användar-ID saknas
+    UserIDWrong: "Begärd användare är inte samma som autentiserad användare"
+    DomainPolicyNil: Organisationspolicy är tom
+    EmailAsUsernameNotAllowed: E-post är inte tillåtet som användarnamn
+    Invalid: Användardata är ogiltig
+    DomainNotAllowedAsUsername: Domänen är redan reserverad och kan inte användas
+    AlreadyInactive: Användaren redan inaktiv
+    NotInactive: Användaren är inte inaktiv
+    CantDeactivateInitial: Användare med status: Initial kan endast raderas, inte avaktiveras
```
[the remark this answers is not in the document]
Comment (JesperWe) [reply internal/static/i18n/sv.yaml]: Fixed

File: internal/static/i18n/sv.yaml (hunk 6 above)
Comment (JesperWe) [reply internal/static/i18n/sv.yaml]: Fixed

File: internal/api/ui/login/static/i18n/sv.yaml (hunk 7 above)
Comment (JesperWe) [reply internal/api/ui/login/static/i18n/sv.yaml]: Fixed

**Review by muhlemmer — APPROVED** [review]
Looks good, thanks!

@JesperWe do you want me to wait merge until @bohemima comes back with the language review?

**Pull request merged**